cli/command/trust/sign_test.go

```diff
@@ -295,3 +295,13 @@ func TestSignCommandChangeListIsCleanedOnError(t *testing.T) {
 	require.NoError(t, err)
 	assert.Equal(t, len(cl.List()), 0)
 }
+
+func TestLocalFlag(t *testing.T) {
```

Comment:
The test name should reflect the function that is being tested, so the name should be something like `TestSignCommandLocalFlag`